arm/Microsoft.ServiceFabric/clusters/.bicep/nested_applications.bicep

```diff
@@ -0,0 +1,24 @@
+param applicationObj object
+param clusterName string
+param location string
+param tags object
+param identity object
+param properties object
+
+resource application 'Microsoft.ServiceFabric/clusters/applications@2021-06-01' = {
+  name: '${clusterName}/application-${applicationObj.name}'
+  location: location
+  tags: tags
+  identity: identity
+  properties: properties
```

Comment:
We should not use a properties object as a such but actually expand its properties to parameters for the corresponding template to allow parameter validation

Reply:
The same is true for all other resources with properties

Reply:
Expanded properties. (Note that for 'applicationTypes' resource type, 'properties' object doesn't contain any properties to set during deployment.)